tests/queries/test_q.py

```diff
@@ -3,6 +3,30 @@
 
 
 class QTests(SimpleTestCase):
+    def test_and_falsey(self):
+        truthy = Q(x=1)
+        combined = truthy & Q()
+        self.assertTrue(combined is truthy)
```

Comment:
I believe you'll want to use `self.assertIs` so there are better error messages, and for consistency with other parts of the test suite.